Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/DatabaseModule.kt

```diff
@@ -0,0 +1,25 @@
+package de.rki.coronawarnapp.storage
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import de.rki.coronawarnapp.storage.tracing.TracingIntervalDao
+import de.rki.coronawarnapp.util.di.AppContext
+
+@Module
+class DatabaseModule {
```

Comment:
Not liking this, smells like `Package by Layer` but we want `Package by feature` :thinking: